### ISSUE TITLE
Issue: Sorry, the content area was not found in your page... 

### ISSUE THREAD

**Issue by JKamphaus** [dudaster/ele-custom-skin#535]
Hi,....

Got a major issue on creating or editing a loop:

#####
“Sorry, the content area was not found in your page. You must call
the_content function in the current template, in order for Elementor to work on this page.”
#####

**How can this be fixed?**

I am using all actuall pro versions

**can´t use Ele custom skin Pro actally**

All the best from Germany

Jörg

**Comment by dudaster** [comment]
Take a Look here: https://github.com/dudaster/ele-custom-skin/issues/201

**Comment by dionisis17** [comment]
> 
> 
> Take a Look here: #201

I got the same problem and the above link with provided solutions can't solve my issue. I can not use Custom Skin Pro at all.
Please help. Thank you very much!

**Comment by dudaster** [comment]
can you make a screenshot of the single, single post instances?

**Comment by dionisis17** [comment]
> 
> 
> can you make a screenshot of the single, single post instances?

![Firefox_Screenshot_2021-07-21T07-43-48 090Z](https://user-images.githubusercontent.com/53856507/126451005-857bbf7b-1e4f-49a3-9aee-d0780222b395.png)
![Firefox_Screenshot_2021-07-21T07-43-24 312Z](https://user-images.githubusercontent.com/53856507/126451007-dc65fdfb-060e-498d-890a-a489fc11169e.png)

The problem just showing in my language...

**Comment by dudaster** [comment]
You have Post template that is set to All singular. Change it to all posts please

**Comment by dionisis17** [comment]
Alright, this problem seems to be fixed. But I got an other issue now when I use the loop
![Firefox_Screenshot_2021-07-21T08-00-50 698Z](https://user-images.githubusercontent.com/53856507/126453355-4f0f1f70-5edd-474e-9bb3-712b098fb6c9.png)
...

**Comment by dudaster** [comment]
this error si because you're using a third party widget inside the loop template that has a dynamic link.

For further info please provide an url so I can take a look at the html code.

**Comment by dionisis17** [comment]
OK. But I have done this thing before, many times, in several sites...Why I get error now?
https://psyctotherm-research.com/ie-e/

**Comment by dudaster** [comment]
I really don't know. WIth other widgets from elementor works fine but with Elementor Addon Elements it seems not to be working.

**Comment by dionisis17** [comment]
> 
> 
> I really don't know. WIth other widgets from elementor works fine but with Elementor Addon Elements it seems not to be working.

But I dont use Elementor Addon Elements. I use (only) happy addons as a 3rd party...And I have never had a problem...

**Comment by dudaster** [comment]
I understand. The issue is that thing that makes the all section a link. And the error comes from elementor. Indeed it is triggered by ECS when the entire template is in a loop instead of single wp_query state. I've tried all the tricks to make it think is inside a single state and it works for all the elementor widgets but for some widgets is not working.

You said you used  this feature in Loop template before? Can you give me an url to take a look? Maybe it worked because you don't have the notice error on. 

What you can do?

Please write an email to the EAE or HAE and kindly ask them to investigate why is this happening with ECS and why the button widget from elementor works just fine.

or

If you have the ECS Pro you can activate an option "Make entire post a link?" to "Yes". Here works because the link is applied directly in the Posts Widget and not inside the Loop template.

**Comment by dionisis17** [comment]
Yes, probably the error didn't shown in my other sites cause I had "Notice error off" 
I will contact to Happy addons and I will inform you...
Thank you very much for all the help. I appreciate it a lot!
